ceph/ceph_admin/dashboard.py

```diff
@@ -112,3 +113,39 @@ def validate_enable_dashboard(cls, username, password):
         LOG.info(
             "Dashboard API login is successful, status code : %s" % resp.status_code
         )
+
+
+def enable_monitoring(cls):
+    """Method to enable the monitoring stack module.
+
+    if user bootstrap with skip-dashboard option
+    then enabling the monitoring stack module.
+
+    Args:
+        cls (CephAdmin object) : cephadm instance object.
+    """
+    out, _ = cls.shell(args=["ceph", "mgr", "services"])
+    formatted = json.loads(out)
+
+    # Getting the ip address to set monitoring stack URL's
+    url = formatted.get("prometheus")
+
+    # Finding IP address using IPV4 format
+    pattern = re.compile(r"(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})")
```

Comment:
Just wanted to know that, are we trying to get active mgr IP here? if yes, can we use `ceph mgr dump` or other command. just to avoid `re`.

Reply:
No ,
Needed Prometheus IP address to enable monitoring stack hence this regex is needed to get that IP address in IPV4 format.

Reply:
Thanks, @Ameenasuhani will try that option.